site/src/docs/components/number-input/code.mdx

```diff
@@ -11,51 +11,56 @@ export default ({ children, pageContext }) => <TabsLayout pageContext={pageConte
 
 ## Code
 
-### Code examples 
+### Code examples
 
 #### Default
-<Playground>
 
+<Playground>
 
 ```jsx
 <>
-<NumberInput
-  helperText="Assistive text"
-  label="Total compensation"
-  unit="€"
-  defaultValue={1000}
-  style={{ maxWidth: '320px' }}
-/>
-
-<NumberInput
-  disabled
-  helperText="Assistive text"
-  label="Total compensation"
-  unit="€"
-  defaultValue={1000}
-  style={{ maxWidth: '320px', marginTop: 'var(--spacing-s)' }}
-/>
+  <NumberInput
+    id="input"
```

Comment:
Looking good! Small improvement idea: The id probably needs to be unique among one html page. Maybe we could give a name that maps with the titles? So this would be `default`, next could be `disabled` etc.

Reply:
Sure, let's do it. I wondered about doing it that way but noticed that the core examples had same ids. Do I change those as well?

Reply:
They could be fixed also. One weird behaviour for same ids is the focus jumping when user clicks the input label.

Reply:
Requested fix has been pushed! Check if this is ok.

Reply:
I think it works now 👍 
They do not even have to be different between React or Core tabs because they are never in the DOM at the same time. 
I was thinking about how realistic some of the ids are, eq. default. Do users realize that they have to replace it with their custom id? Maybe they will 🤔

Reply:
It's sort of a silly name for an id, not very realistic, but it relates to the context in documentation. I think the id names are fine but it wouldn't be a bad idea to include accessibility information about label for input id somewhere? At a quick glance I didn't find anything about it in NumberInput or TextField component documentation or even at Patterns/Form building section.